recipe_sha: 6c96e2949fdfd6eb24775e06ac4830cc4f34935fca5b8b0c2b36cb9095c2f688  # The commit at which this recipe failed to build.
skiplist: true # Set to true to skiplist this recipe so that it will be ignored as long as its latest commit is the one given above.
log: |-
  - Encountered problems while solving:
  -   - nothing provides requested bioconductor-reactomepa >=1.44.0,<1.45.0
  -

  Leaving build/test directories:
    Work:
   /opt/conda/conda-bld/work
    Test:
   /opt/conda/conda-bld/test_tmp
  Leaving build/test environments:
    Test:
  source activate  /opt/conda/conda-bld/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl
    Build:
  source activate  /opt/conda/conda-bld/_build_env


  Traceback (most recent call last):
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 141, in mamba_get_install_actions
      solution = solver.solve_for_action(_specs, prefix)
    File "/opt/conda/lib/python3.8/site-packages/boa/core/solver.py", line 244, in solve_for_action
      t = self.solve(specs)
    File "/opt/conda/lib/python3.8/site-packages/boa/core/solver.py", line 234, in solve
      raise RuntimeError("Solver could not find solution."  error_string)
  RuntimeError: Solver could not find solution.Mamba failed to solve:
   - r-rmarkdown
   - r-visnetwork
   - r-outliers
   - bioconductor-s4vectors >=0.38.0,<0.39.0
   - r-nntensor >=1.1.5
   - r-plotly
   - r-abind
   - bioconductor-meshr >=2.6.0,<2.7.0
   - bioconductor-biocstyle >=2.28.0,<2.29.0
   - r-knitr
   - bioconductor-annotationhub >=3.8.0,<3.9.0
   - bioconductor-meshdbi >=1.36.0,<1.37.0
   - bioconductor-annotationdbi >=1.62.0,<1.63.0
   - r-tagcloud
   - r-checkmate
   - r-rtensor >=1.4.8
   - r-base 4.3.*
   - bioconductor-category >=2.66.0,<2.67.0
   - r-igraph
   - r-ggplot2
   - bioconductor-summarizedexperiment >=1.30.0,<1.31.0
   - r-cctensor >=1.0.2
   - r-plotrix
   - bioconductor-schex >=1.14.0,<1.15.0
   - bioconductor-gostats >=2.66.0,<2.67.0
   - r-heatmaply
   - r-biocmanager
   - bioconductor-reactomepa >=1.44.0,<1.45.0
   - r-crayon
   - r-rsqlite
   - bioconductor-singlecellexperiment >=1.22.0,<1.23.0
   - bioconductor-reactome.db >=1.84.0,<1.85.0
   - bioconductor-dose >=3.26.0,<3.27.0

  with channels:
   - conda-forge
   - bioconda
   - defaults

  The reported errors are:
  - Encountered problems while solving:
  -   - nothing provides requested bioconductor-reactomepa >=1.44.0,<1.45.0
  -

  During handling of the above exception, another exception occurred:

  Traceback (most recent call last):
    File "/opt/conda/bin/conda-mambabuild", line 10, in <module>
      sys.exit(main())
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 256, in main
      call_conda_build(action, config)
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 228, in call_conda_build
      result = api.build(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/api.py", line 180, in build
      return build_tree(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 3078, in build_tree
      packages_from_this = build(metadata, stats,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 2038, in build
      output_metas = expand_outputs([(m, need_source_download, need_reparse_in_env)])
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 787, in expand_outputs
      for (output_dict, m) in deepcopy(_m).get_output_metadata_set(permit_unsatisfiable_variants=False):
    File "/opt/conda/lib/python3.8/site-packages/conda_build/metadata.py", line 2524, in get_output_metadata_set
      conda_packages = finalize_outputs_pass(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/metadata.py", line 884, in finalize_outputs_pass
      fm = finalize_metadata(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 547, in finalize_metadata
      build_unsat, host_unsat = add_upstream_pins(m,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 409, in add_upstream_pins
      host_deps, host_unsat, extra_run_specs_from_host = _read_upstream_pin_files(m, 'host',
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 374, in _read_upstream_pin_files
      deps, actions, unsat = get_env_dependencies(m, env, m.config.variant,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 131, in get_env_dependencies
      actions = environ.get_install_actions(tmpdir, tuple(dependencies), env,
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 150, in mamba_get_install_actions
      raise err
  conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {MatchSpec("bioconductor-reactomepa[version='>=1.44.0,<1.45.0']")}
# Last 100 lines of the build log.
